client/src/partials/templates/messages.tmpl.html

```diff
@@ -0,0 +1,23 @@
+<!-- displays "This value is too small for this field." -->
+<p ng-message="min">This value is too small for this field.</p>
```

Comment:
If it will not take very long. There's no reason at this stage in 
development it should be blocking.

On 09/02/2016 11:55, Jonathan Niles wrote:
>
> In client/src/partials/templates/messages.tmpl.html 
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/89#discussion_r52292848>:
>
> > @@ -0,0 +1,23 @@
> > +<!-- displays "This value is too small for this field." -->
> > +<p ng-message="min">This value is too small for this field.</p>
>
> :+1: Would you like me to include translations in this PR?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/IMA-WorldHealth/bhima-2.X/pull/89/files#r52292848>.
>